### PR TITLE
generate:  support adding additional mounts for container

### PR DESCRIPTION
**Pull request by Mashimiao**
Add support for adding additional mounts.
Users can add custom additional mounts, like adding tmpfs without special options or with custom options.

### CODE REVIEW

**Comment by Mashimiao** [comment]
Rebased.  @opencontainers/runtime-tools-maintainers  PTAL

File: cmd/oci-runtime-tool/generate.go [hunk 1]
```diff
@@ -896,6 +908,23 @@ func parseBindMount(s string) (string, string, []string, error) {
...
+	mparts := strings.SplitN(s, ":", 4)
+	switch len(mparts) {
+	case 3:
+		source, dest, mType = mparts[0], mparts[1], mparts[2]
+	case 4:
+		source, dest, mType, options = mparts[0], mparts[1], mparts[2], strings.Split(mparts[3], ":")
+	default:
+		return source, dest, mType, options, fmt.Errorf("--mounts-add should have format src:dest:type:[:options...]")
+	}
+
```
Comment (q384566678) [inline cmd/oci-runtime-tool/generate.go]: The non-null verification of `dest` should be increased because it is required.

**Comment by q384566678** [comment]
LGTM

**Comment by liangchenye** [comment]
decoupled with #439, I'll merge it.

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**